include/CommandHandler/src/CommandHandler.cpp

```diff
@@ -606,19 +618,22 @@ void CommandHandler::ReadData(SocketWrapper& socket_wrapper, std::string& data_m
     Logger::LogDebug("Exiting CommandHandler::ReadData");
 }
 
-void CommandHandler::ProcessDataMessage(SocketWrapper& socket_wrapper,
-                                    std::string& data_message)
+void CommandHandler::ProcessDataMessage(SocketWrapper& socket_wrapper, std::string& data_message)
 {
     Logger::LogDebug("Exiting CommandHandler::ProcessDataMessage");
-    Logger::LogTrace("CommandHandler::ProcessDataMessage parameters: "
-                     "SocketWrapper reference, std::string reference " + data_message);
+    Logger::LogTrace(
+        "CommandHandler::ProcessDataMessage parameters: "
+        "SocketWrapper reference, std::string reference " +
+        data_message);
 
     std::size_t pos;
-    while ((pos = data_message.find("\r\n")) != std::string::npos) {
+    while ((pos = data_message.find("\r\n")) != std::string::npos)
+    {
         std::string line = data_message.substr(0, pos);
-        data_message.erase(0, pos + 2);
+        data_message.erase(0, pos + DELIMITER_OFFSET);
 
-        if (line == ".") {
+        if (line == ".")
+        {
             Logger::LogProd("End-of-data sequence detected, exiting data read loop.");
```

Comment:
You find end of data twice: once in `ReadData` and in `ProcessDataMessage`. Maybe this two function can be merged.